breathecode/assignments/tests/urls/tests_user_me_project.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test /final_project/<int:project_id>
+"""
+from django.utils import timezone
+from datetime import timedelta
+from unittest.mock import MagicMock, call, patch
+
+from django.urls.base import reverse_lazy
+from rest_framework import status
+
+from ..mixins import AssignmentsTestCase
+
+
+class FinalProjectTestSuite(AssignmentsTestCase):
+    """Test /final_project"""
+    """
+    🔽🔽🔽 Auth
+    """
+
+    def test_final_project_with_no_auth(self):
```

Comment:
Some validation like this https://github.com/gustavomm19/apiv2/blob/b786f2d770bb736c9075762c921b1e6bec6ca04e/breathecode/assignments/serializers.py#L306 was not cover